ntoskrnl/ke/i386/cpu.c

```diff
@@ -1400,7 +1400,7 @@ KeRestoreFloatingPointState(IN PKFLOATING_SAVE Save)
 {
     PFNSAVE_FORMAT FpState = *((PVOID *) Save);
     ASSERT(KeGetCurrentThread()->Header.NpxIrql == KeGetCurrentIrql());
-    DPRINT1("%s is not really implemented\n", __FUNCTION__);
+    DPRINT("%s is not really implemented\n", __FUNCTION__);
```

Comment:
May be eligible for `UNIMPLEMENTED_ONCE`.

Reply:
Yes please

Reply:
Unless someone disagrees, i will fix this in the evening after work.
Only for Ke*FloatingPointState or for the other 2 as well?

Reply:
I think it would be OK for KeSave/RestoreFloatingPointState and SrvGetThreadConsoleDesktop.
I'm not sure about MiInsertVadEx() though....

Reply:
maybe @tkreuzer or @HeisSpiter have an opinion about this?